docs/application/dotnet/guides/nui/text.md

```diff
@@ -401,6 +401,23 @@ The following example illustrates text alignment:
 field.HorizontalAlignment = HorizontalAlignment.Begin;
 ```
 
+<a name="input-properties"></a>
+### Use Input Properties in TextField
+
+To change the text settings for new input text, you can use the Input properties of the [Tizen.NUI.BaseComponents.TextField](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.TextField.html) class, such as `InputColor`, `InputPointSize`, and so on.
+
+It is recommended to use these properties with some specific conditions. For instance, you can use these properties either when the `TextChanged` event occurs or when the `Clicked` event of Button occurs.

```

Comment:
or when the `Clicked` event of the button occurs.

Reply:
Done! :)